app/src/main/java/org/wikipedia/places/PlacesFilterActivity.kt

```diff
@@ -54,6 +54,9 @@ class PlacesFilterActivity : BaseActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityPlacesFiltersBinding.inflate(layoutInflater)
         setUpRecyclerView()
+        setStatusBarColor(ResourceUtil.getThemedColor(this, R.attr.background_color))
+        setNavigationBarColor(ResourceUtil.getThemedColor(this, R.attr.background_color))
+        supportActionBar?.setBackgroundDrawable(ColorDrawable(ResourceUtil.getThemedColor(this, R.attr.background_color)))
```

Comment:
To have full control of the style and color of a Toolbar, an activity should have its own explicit Toolbar in its layout (i.e. not using `style/AppTheme.ActionBar`).